sccm/protect/deploy-use/endpoint-antimalware-policies.md

```diff
@@ -204,6 +204,9 @@ Cloud Protection Service enables the collection of information about detected ma
 
 **Allow extended cloud check to block and scan for up to (seconds)** - Specifies the number of seconds Cloud Protection Service can block a file while the service checks that the file is not known to be malicious.
 
+> [!NOTE]  
+> The number of seconds selected for this setting will be in addition to a 10-second timeout built in to the product.  Example: if you enter 0 seconds, CSP will be able to lock the file for up to 10 seconds.
```

Comment:
A rewording suggestion for style:

```suggestion
> The number of seconds that you select for this setting is in addition to a default 10-second timeout. For example, if you enter 0 seconds, the Cloud Protection Service blocks the file for 10 seconds.
```

Reply:
Thanks, yours definitely sounds much better.